TPP.Core/Modes/ModeBase.cs

```diff
@@ -133,6 +134,11 @@ private async Task ProcessIncomingMessage(IChat chat, Message message)
             {
                 return;
             }
+            if (message.User.Banned || message.User.TimeoutExpiration > now)
+            {
+                _logger.LogDebug("Skipping message from {User} because user is banned or timed out", message.User);
+                return;
+            }
```

Comment:
fyi, this implies that these messages also are not forwarded to old core. So any banned or timed out user is cut off (only affects whispers since they wouldn't be able to write in chat anyway)